webapp/channels/src/utils/utils.tsx

```diff
@@ -90,6 +90,23 @@ const CLICKABLE_ELEMENTS = [
     'audio',
     'video',
 ];
+const MS_PER_SECOND = 1000;
+const MS_PER_MINUTE = 60 * MS_PER_SECOND;
+const MS_PER_HOUR = 60 * MS_PER_MINUTE;
+const MS_PER_DAY = 24 * MS_PER_HOUR;
+
+export enum TimeInformation {
+    MILLISECONDS = 'm',
+    SECONDS = 's',
+    MINUTES = 'x',
+    HOURS = 'h',
+    DAYS = 'd',
+    FUTURE = 'f',
+    PAST = 'p'
+}
+
+export type TimeUnit = Exclude<TimeInformation, TimeInformation.FUTURE | TimeInformation.PAST>;
+export type TimeDirection = TimeInformation.FUTURE | TimeInformation.PAST;
```

Comment:
nit: What about defining `TimeUnit` and `TimeDirection` directly, and import them where needed?

Reply:
I rather to keep as it is right now, TimeInformation is an enum which is intended to contain this types and maybe more if needed and TimeUnit and TimeDirection are intended to be just types.